roles/kubernetes/preinstall/tasks/0020-verify-settings.yml

```
@@ -183,6 +183,13 @@
     msg: "kube_pods_subnet cannot be the same network segment as kube_service_addresses"
   run_once: yes
 
+- name: "Check that IP range is enough for the nodes"
+  assert:
+    that:
+      - 2 ** ( kube_network_node_prefix - kube_pods_subnet | ipaddr('prefix' ) >= groups['k8s_cluster'] | length
```

I'm guessing you are missing a parentheses.

```suggestion
      - 2 ** ( kube_network_node_prefix - kube_pods_subnet | ipaddr('prefix' ) >= groups['k8s_cluster'] | length )
```

---

Hello, thank you for your review! Basic mistakes for the first start🥲